docs/web_sdk/Changelog.md

```diff
@@ -2,6 +2,88 @@
+  - New promotion badge on recipe cards
+  - New promotions banner on catalog page, it is hidden by default and must be displayed manually
+- *recipe-details*
+  - Displayed a "Last Order" badge showing the date of the user's most recent order (if applicable)
+  - Added the logic for the "Last Order" modal
+  - Added a banner under the recipe steps in the "Cooking" tab to allow users to reorder previously ordered recipes
+  - Can now replace unavailable products
+- *last-order-modal*
+  - Created a component triggered by clicking the "Last Order" badge, allowing users to reorder recipes not currently in their basket
+- *basket-preview*
+  - Add footer to redirect to client's cart
+- *analytics*:
+  - New basket.display event sent on click on "X recipes in your basket" in catalog
+  - New home.display event sent on click on "See more recipes" in recipe details
+  - New recipe.shopping.display event sent on click on "Shopping" in recipe details
+  - New recipe.coooking.display event sent on click on "Cooking" in recipe details
+  - New entry.ignore event sent on click on "Ignore" in product card
+  - New entry.replace event sent on click on "Replace" or "Choose a product" in product card
+- *window.mealz.basket.openPreview*:
+  - New `analyticsPath` property for analytics
+- *window.mealz.user*:
+  - Added a method `loadWithAuthlessId: (id: string, forbidProfiling?: boolean) => void` to inform Mealz of the authless-id fetched with the SSR route /generate-authless-token
+  - Added a method `updateForbidProfiling: (forbidProfiling: boolean, userId: string, isAuthless: boolean) => void` to inform Mealz that the profiling permission has changed during execution. It can be used for both logged mode and authless mode (ex: logged user : `updateForbidProfiling(true, 'user-id')` | authless user: `updateForbidProfiling(true, 'MEALZ-AUTHLESS-12345', true)`)
+
+#### Updated:
+- *catalog-header*
+  - Header label changes to "My space" when history is enabled; otherwise, it remains "Favorites"
+- *catalog-toolbar*
+  - "Filters" and "Preferences" buttons are now hidden in the "Favorites" page
+  - "Filters" and "Preferences" are not applied in the "Favorites" page
+  - Button label changes to "My space" when history is enabled; otherwise, it remains "Favorites"
+- *recipe-catalog*
+  - Text changes to "My space" when history is enabled; otherwise, it remains "Favorites"
+  - Display "Favorites" and "History" tabs when history is enabled
+  - History content is shown only when order history is enabled.
+- *recipe-details*
+  - Removed the footer with the CTA "Add X products (XX,XX€)" in the "Cooking" tab
+- *products-picker*:
+  - Unavailable products now display into the default section
+- *product-card*:
+  - New view for unavailable state
+- *modal*
+- Adjusted CSS using ">" selectors to prevent styles from affecting multiple simultaneously displayed modals
+- Removed the footer with the CTA "Add X products (XX,XX€)" in the "Cooking" tab
```

Comment:
```suggestion
  - Adjusted CSS using ">" selectors to prevent styles from affecting multiple simultaneously displayed modals
  - Removed the footer with the CTA "Add X products (XX,XX€)" in the "Cooking" tab
```